### PR TITLE
Add commas section to general styleguide

### PR DESCRIPTION
**Pull request by eb-namchi**
screen shot 2017-06-20 at 10 25 33 am

### CODE REVIEW

**Comment by gagoar** [comment]
LGTM, shouldn't you say that this is with es6 ? (even tho you are using `const`, but still)

**Comment by kaylieEB** [comment]
can you link to eslint page? http://eslint.org/docs/rules/comma-dangle#always-multiline. Also I'd just remove (we'll be running...etc) so you don't have to update the docs again.

File: general/README.md [hunk 1]
```diff
@@ -538,3 +539,43 @@ UpdateAll: function() {
 ```
 
 **[⬆ back to top](#table-of-contents)**
+
+## Commas
+
+Trailing Commas are only enforced for multi-line objects and arrays (we'll be running a codemod to fix previous issues, for now you can add them if they're multiline).
```
Comment (albybarber) [inline general/README.md]: I think you can remove the mention to the code mod, we can just do it.

Comment (albybarber) [inline general/README.md]: Also lets add the `es6lint` link that @kaylieEB suggested

**Comment by albybarber** [comment]
🚢

**Pull request merged**